lib/tapjoy/ldap/key/install.rb

```diff
@@ -0,0 +1,46 @@
+module Tapjoy
+  module LDAP
+    module Key
+      # Install key on localhost
+      class Install
+        def install
+          Tapjoy::LDAP::Key.get_keys_from_ldap.each do |user, values|
+            directory = directory(user)
+            FileUtils.mkdir_p(directory) unless File.exists?directory
+            authorized_keys_file = "#{directory}/authorized_keys"
+            keys = load_keys_from_file(authorized_keys_file)
+            insert_keys(authorized_keys_file, keys, values)
+          end
+        end
+
+        private
+        def opts
+          @opts ||= Trollop::options do
+              # Set help message
+              usage 'key install [options]'
```

Comment:
actually do we want to add an optional user option here so we don't have to install all the ldap keys on localhost when this is run? I can see the use for downloading all the keys during a bootstrap operation, but we may have cases where we don't want all the keys.  This is more food for thought than something that would have to be part of the scope of this pr.